wordpress-mobile/WordPress-iOS
# Improve calculation of media directories.

Include in the calculation of the media cache the local media cache for fullsizes images that are uploaded on the device.

Fixes #12306

To test:
 - Open the app.
 - Open the Media library of one site.
 - Add one media to the media library.
 - Go to Me -> App Settings
 - Check the size of the media cache
 - Check  the size reported by iOS in Settings -> iPhone Storage -> WordPress
 - Press the Clear Device Media Cache option in the app
 - Check again in Settings to see if the size decrease match the amount of cache cleaned.


Update release notes:

- [ x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.
